api/types/types.go

```diff
@@ -9,3 +9,13 @@ type ContainerCreateResponse struct {
 	// Warnings are any warnings encountered during the creation of the container.
 	Warnings []string `json:"Warnings"`
 }
+
+// ContainerExecCreateResponse contains the information returned to a client on the
+// execution of a command in a container.
```

Comment:
This comment is actually not fully true, because `execCreate` not executes anything in container, it just creates internal datastructure for later start.